browser-test/src/applicant/application_version_fast_forward.test.ts

```diff
@@ -486,6 +487,503 @@ test.describe('Application Version Fast-Forward Flow', () => {
       })
     })
   })
+
+  test('all major steps - fast forward flag enabled', async ({browser}) => {
```

Comment:
This is a slightly modified version of the disabled version. Guess there's not a good way to show the diff since I'm keeping the feature flag disabled version for now.